### DISCUSSION TITLE
Server Issue

### DISCUSSION THREAD

**Discussion by git404x** [ROM-builders/temporary#13682] [General]
https://cirrus-ci.com/build/6359154800459776 

this build still not test passed from more than 30 min

https://cirrus-ci.com/build/6085892119986176

here only push user able to build any solution i am getting checkout error

**Comment by git404x** [comment]
what happened to server ??

**Reply by Apon77** [reply]
I have no idea why this is happening

**Reply by git404x** [reply]
ok
